src/pages/Lending/AvailablePools.tsx

```diff
@@ -62,6 +64,24 @@ const LendingPoolButton = ({ asset, onPoolClick }: LendingPoolButtonProps) => {
     [isLendingPositionDataLoading, isPoolDataLoading],
   )
 
+  const StatusTag = useCallback(() => {
+    if (poolData?.status === 'Available') {
+      return (
+        <Tag colorScheme='green'>
+          <TagLeftIcon as={CheckCircleIcon} />
+          {translate('common.active')}
+        </Tag>
+      )
+    }
+
+    return (
+      <Tag colorScheme='red'>
+        <TagLeftIcon as={BiErrorCircle} />
+        {translate('common.halted')}
+      </Tag>
+    )
+  }, [poolData?.status, translate])
```

Comment:
q: as we've previously discussed, do we want to surface the caps full info here as well? i.e a pool may be active, but unactionable for borrows as you've mentioned, which may be weird for users, as they will see "Active" but then a "halted" messaging in borrow confirm

<img width="1446" alt="image" src="https://github.com/shapeshift/web/assets/17035424/f396cbc0-98df-4ab9-bd6d-336b0e3784d0">

suggestion: while at it, we may also want to improve the "halted" messaging to reflect caps full more accurately